### PR TITLE
Add more things to main branch

### PR DESCRIPTION
**Pull request by britaumich**

### CODE REVIEW

**Review by jocelynanderson — DISMISSED** [review]
Ordering and additions to emails look good!

**Review by jocelynanderson — APPROVED** [review]

File: app/views/mailers/reservation_mailer/car_reservation_approved.html.erb [hunk 1]
```diff
@@ -43,5 +43,13 @@
       To view/modify/cancel reservations, click here
       <%= link_to "LSA RideShare", welcome_pages_student_url %>
     </p>
+
+    <p>
+      <span style="color:red; font-weight:bold">Vehicle problems or breakdowns?</span>
+      <ul>
+        <li>Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490</li>
```
Comment (anantaUM) [inline app/views/mailers/reservation_mailer/car_reservation_approved.html.erb:50]: is this meant to be 12:30 _am_ or _pm_ ? Also, can you please specify _am_ next to 8:00.

File: app/views/mailers/reservation_mailer/car_reservation_approved.text.erb [hunk 2]
```diff
@@ -28,3 +28,7 @@ You can email <%= @unit_email %> or call the office at: <%= @contact_phone %>.
 
 To view/modify/cancel reservations, click here
 <%= link_to "LSA RideShare",  welcome_pages_student_url%>
+
+Vehicle problems or breakdowns?
+Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490
```
Comment (anantaUM) [inline app/views/mailers/reservation_mailer/car_reservation_approved.text.erb:33]: same review comment which is mentioned in _car_reservation_approved.html.erb_

File: app/views/mailers/reservation_mailer/car_reservation_confirmation.html.erb [hunk 3]
```diff
@@ -50,5 +50,13 @@
       To view/modify/cancel reservations, click here
       <%= link_to "LSA RideShare", welcome_pages_student_url %>
     </p>
+
+    <p>
+      <span style="color:red; font-weight:bold">Vehicle problems or breakdowns?</span>
+      <ul>
+        <li>Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490</li>
```
Comment (anantaUM) [inline app/views/mailers/reservation_mailer/car_reservation_confirmation.html.erb:57]: same review comment which is mentioned in _car_reservation_approved.html.erb_

File: app/views/mailers/reservation_mailer/car_reservation_confirmation.text.erb [hunk 4]
```diff
@@ -30,3 +30,7 @@ You can email <%= @unit_email %> or call the office at: <%= @contact_phone %>.
 
 To view/modify/cancel reservations, click here
 <%= link_to "LSA RideShare",  welcome_pages_student_url %>
+
+Vehicle problems or breakdowns?
+Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490
```
Comment (anantaUM) [inline app/views/mailers/reservation_mailer/car_reservation_confirmation.text.erb:35]: same review comment which is mentioned in _car_reservation_approved.html.erb_

File: app/views/mailers/reservation_mailer/car_reservation_drivers_edited.html.erb [hunk 5]
```diff
@@ -43,5 +43,13 @@
       To view/modify/cancel reservations, click here
       <%= link_to "LSA RideShare", welcome_pages_student_url %>
     </p>
+
+    <p>
+      <span style="color:red; font-weight:bold">Vehicle problems or breakdowns?</span>
+      <ul>
+        <li>Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490</li>
```
Comment (anantaUM) [inline app/views/mailers/reservation_mailer/car_reservation_drivers_edited.html.erb:50]: same review comment which is mentioned in _car_reservation_approved.html.erb_

File: app/views/mailers/reservation_mailer/car_reservation_drivers_edited.text.erb [hunk 6]
```diff
@@ -27,3 +27,7 @@ You can email <%= @unit_email %> or call the office at: <%= @contact_phone %>.
 
 To view/modify/cancel reservations, click here
 <%= link_to "LSA RideShare",  welcome_pages_student_url%>
+
+Vehicle problems or breakdowns?
+Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490
```
Comment (anantaUM) [inline app/views/mailers/reservation_mailer/car_reservation_drivers_edited.text.erb:32]: same review comment which is mentioned in _car_reservation_approved.html.erb_

File: app/views/mailers/reservation_mailer/car_reservation_update_passengers.html.erb [hunk 7]
```diff
@@ -40,5 +40,13 @@
       To view/modify/cancel reservations, click here
       <%= link_to "LSA RideShare", welcome_pages_student_url %>
     </p>
+
+    <p>
+      <span style="color:red; font-weight:bold">Vehicle problems or breakdowns?</span>
+      <ul>
+        <li>Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490</li>
```
Comment (anantaUM) [inline app/views/mailers/reservation_mailer/car_reservation_update_passengers.html.erb:47]: same review comment which is mentioned in _car_reservation_approved.html.erb_

File: app/views/mailers/reservation_mailer/car_reservation_update_passengers.text.erb [hunk 8]
```diff
@@ -25,3 +25,7 @@ You can email <%= @unit_email %> or call the office at: <%= @contact_phone %>.
 
 To view/modify/cancel reservations, click here
 <%= link_to "LSA RideShare",  welcome_pages_student_url%>
+
+Vehicle problems or breakdowns?
+Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490
```
Comment (anantaUM) [inline app/views/mailers/reservation_mailer/car_reservation_update_passengers.text.erb:30]: same review comment which is mentioned in _car_reservation_approved.html.erb_

File: app/views/mailers/reservation_mailer/car_reservation_updated.html.erb [hunk 9]
```diff
@@ -43,5 +43,12 @@
       To view/modify/cancel reservations, click here
       <%= link_to "LSA RideShare", welcome_pages_student_url %>
     </p>
+    <p>
+      <span style="color:red; font-weight:bold">Vehicle problems or breakdowns?</span>
+      <ul>
+        <li>Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490</li>
```
Comment (anantaUM) [inline app/views/mailers/reservation_mailer/car_reservation_updated.html.erb:49]: same review comment which is mentioned in _car_reservation_approved.html.erb_

File: app/views/mailers/reservation_mailer/car_reservation_updated.text.erb [hunk 10]
```diff
@@ -23,3 +23,7 @@ You can email <%= @unit_email %> or call the office at: <%= @contact_phone %>.
 
 To view/modify/cancel reservations, click here
 <%= link_to "LSA RideShare",  welcome_pages_student_url %>
+
+Vehicle problems or breakdowns?
+Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490
```
Comment (anantaUM) [inline app/views/mailers/reservation_mailer/car_reservation_updated.text.erb:28]: same review comment which is mentioned in _car_reservation_approved.html.erb_

**Review by anantaUM — CHANGES_REQUESTED** [review]

**Comment by anantaUM** [comment]
@britaumich - The V_ehicle problems or breakdowns?_ timings are repeated in 10 files. It violates the DRY principle. Could you please make changes such that it is defined in one place and used in rest of the files ? 
Thanks.

**Review by jocelynanderson — APPROVED** [review]

**Review by jocelynanderson — APPROVED** [review]

**Comment by britaumich** [comment]
I edited email messages. First I just copied Sarah's message. Now I have official information about Garage Services. It's 7:00 a.m. - 12:30 a.m.
This message is not just a variable it's a formatted HTML text. I am not sure how to put it into a variable. Also, helpers are not available in the mailer files.  
I'll think about how to refactor this code later.

**Review by jocelynanderson — APPROVED** [review]

File: app/views/mailers/reservation_mailer/car_reservation_approved.html.erb (hunk 1 above)
Comment (britaumich) [reply app/views/mailers/reservation_mailer/car_reservation_approved.html.erb]: time fixed, code moved to partial

File: app/views/mailers/reservation_mailer/car_reservation_confirmation.html.erb (hunk 3 above)
Comment (britaumich) [reply app/views/mailers/reservation_mailer/car_reservation_confirmation.html.erb]: time fixed, code moved to partial

File: app/views/mailers/reservation_mailer/car_reservation_confirmation.text.erb (hunk 4 above)
Comment (britaumich) [reply app/views/mailers/reservation_mailer/car_reservation_confirmation.text.erb]: time fixed

File: app/views/mailers/reservation_mailer/car_reservation_drivers_edited.html.erb (hunk 5 above)
Comment (britaumich) [reply app/views/mailers/reservation_mailer/car_reservation_drivers_edited.html.erb]: time fixed, code moved to partial

File: app/views/mailers/reservation_mailer/car_reservation_drivers_edited.text.erb (hunk 6 above)
Comment (britaumich) [reply app/views/mailers/reservation_mailer/car_reservation_drivers_edited.text.erb]: time fixed

File: app/views/mailers/reservation_mailer/car_reservation_update_passengers.html.erb (hunk 7 above)
Comment (britaumich) [reply app/views/mailers/reservation_mailer/car_reservation_update_passengers.html.erb]: time fixed, code moved to partial

File: app/views/mailers/reservation_mailer/car_reservation_update_passengers.text.erb (hunk 8 above)
Comment (britaumich) [reply app/views/mailers/reservation_mailer/car_reservation_update_passengers.text.erb]: time fixed

File: app/views/mailers/reservation_mailer/car_reservation_updated.html.erb (hunk 9 above)
Comment (britaumich) [reply app/views/mailers/reservation_mailer/car_reservation_updated.html.erb]: time fixed, code moved to partial

File: app/views/mailers/reservation_mailer/car_reservation_updated.text.erb (hunk 10 above)
Comment (britaumich) [reply app/views/mailers/reservation_mailer/car_reservation_updated.text.erb]: time fixed

**Comment by britaumich** [comment]
Here is a Jira ticket to research how to render partial in a text email template:
https://umlsait.atlassian.net/browse/LRA-703

**Review by anantaUM — APPROVED** [review]

**Pull request merged**